.pre-commit-config.yaml

```diff
@@ -250,7 +250,7 @@ repos:
         exclude: |
           (?x)
           ^airflow/_vendor/
-  - repo: https://github.com/ikamensh/flynt/
+  - repo: https://github.com/ikamensh/flynt
```

Comment:
This is a local copy of https://github.com/apache/airflow/pull/18151 to let me get the commit in; it will vanish upon rebase when that lands.